js-service/extras/common.js

```diff
@@ -0,0 +1,38 @@
+(function () {
+    'use strict';
+
+    angular
```

Comment:
this is a constant file....it can be created as below...pls replace service with constants

(function () {
    'use strict';

    angular
        .module('flairbiApp')
        .constant('scheduler_visualizations', [
            'Pie',
            'Line'
            ]
        );
})();